mediaire_toolbox/transaction_db/transaction_db.py

```diff
@@ -670,8 +689,11 @@ def get_user_preferences(self, user_id: int) -> dict:
             self.session.rollback()
 
     @t_db_retry
-    def add_study_metadata(self, study_id: str, origin: str,
-                           c_move_time: datetime, overwrite: bool=False):
+    def add_study_metadata(self,
+                           study_id: str,
+                           origin: str,
+                           c_move_time: datetime,
```

Comment:
we have to make sure that we use timezone aware objects for this function in our other projects. see https://gitlab.com/mediaire/longitudinal_grazer/-/blob/master/longitudinal_grazer/longitudinal_grazer.py#L441
and https://gitlab.com/mediaire/dicom_grazer/-/blob/master/dicom_grazer/dicom_grazer.py#L417

Reply:
done
https://gitlab.com/mediaire/dicom_grazer/-/merge_requests/47/diffs?commit_id=b7ac995f9d55093ebad9287af53b927f6e298bf1
https://gitlab.com/mediaire/longitudinal_grazer/-/merge_requests/23/diffs?commit_id=3f4c2afcb1e41de7f326bb0821f0047cb0b298f4